### PR TITLE
Maestro cleanup

### PR DESCRIPTION
**Pull request by mray190**
Not urgent
Removes un-used maestro features like mdns, templating, devicedb, and logging

### CODE REVIEW

File: maestro/rpm/files/pelion-base-config.yaml [hunk 1]
```diff
@@ -8,10 +8,6 @@ scratchPath: /tmp/maestro/scratch
 clientId: "{{ARCH_SERIAL_NUMBER}}"
 network:
   disable: true
-platform_readers:
-  - platform: "fsonly"
-    params:
-      identityPath: "/var/lib/pelion/edge_gw_config/identity.json"
```
Comment (kbgl) [inline maestro/rpm/files/pelion-base-config.yaml:14]: Is `identity.json` still needed for anything? `wait-for-pelion-identity.service` blocks till this file is created.

**Comment by mray190** [comment]
Ready to go. CI failure can be ignored - all our github action PR checks pass and build

File: README.md [hunk 2]
```diff
@@ -522,7 +522,6 @@ installed simultaneously.
 
 4. Enable `systemd` services. After installation there are following services:
 ```
-devicedb.service
```
Comment (costanic) [inline README.md:525]: do we need to add fluent-bit to this list?

File: README.md [hunk 3]
```diff
@@ -534,7 +533,7 @@ wait-for-pelion-identity.service
 
 To enable all services, run:
 ```bash
-sudo systemctl enable devicedb.service edge-core.service edge-proxy.service kubelet.service maestro.service mbed-fcc.service pelion-relay-term.service wait-for-pelion-identity.service
```
Comment (costanic) [inline README.md:537]: do we need to add fluent-bit here?

File: build-env/target/rhel/8/packages.conf.sh [hunk 4]
```diff
@@ -6,7 +6,6 @@ DEPENDS=(
 
 PACKAGES=(
     'pe-nodejs'
-    'devicedb'
```
Comment (costanic) [inline build-env/target/rhel/8/packages.conf.sh:9]: add fluent-bit here?

**Review by johnmichaelwelsh — APPROVED** [review]
Looks good. Does it work?

File: build-env/target/rhel/8/packages.conf.sh (hunk 4 above)
Comment (mray190) [reply build-env/target/rhel/8/packages.conf.sh:9]: @costanic we currently do not support fluent-bit in rhel 8 - only debian packages.
Once we create a fluent-bit.spec for rhel, we can add it here

**Comment by mray190** [comment]
> Looks good. Does it work?

Yep it works! Devicedb and Maestro-shell no longer run, and maestro still runs without erroring

**Pull request merged**